<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1566</width>
    <height>923</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1566</width>
    <height>923</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1566</width>
    <height>923</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>153</y>
      <width>1191</width>
      <height>751</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QComboBox" name="departure">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>169</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Departure</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Busan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HochiMinh</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LongBeach</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>133</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Departure, Arrival</string>
    </property>
   </widget>
   <widget class="QPushButton" name="applybutton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Find Route</string>
    </property>
    <widget class="QPushButton" name="start_button">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>27</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕 Semilight</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
image:url(D:/Optimizer_route/jMetalPy-master/examples/singleobjective/nsgaii/start..png)
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>301</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>MapControl</string>
    </property>
    <widget class="QPushButton" name="loadmap">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>27</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
image:url(D:/Optimizer_route/jMetalPy-master/examples/singleobjective/nsgaii/mapcontrol.png)
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>48</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Load Map</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadwheather">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>27</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
image:url(D:/Optimizer_route/jMetalPy-master/examples/singleobjective/nsgaii/wheathercontrol.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>181</x>
       <y>90</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Load wheather</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>591</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Algorithm Status</string>
    </property>
    <widget class="QLineEdit" name="Node_line">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>38</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>59</x>
       <y>34</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Node</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>41</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Max evaluations</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="maxevaluations">
     <property name="geometry">
      <rect>
       <x>298</x>
       <y>39</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="populationsize">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>70</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>223</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>lower bound</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>399</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>upper bound</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lowerbound">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>72</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>population size</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="upperbound">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>10</y>
      <width>271</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Optimization Result</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>37</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Distance :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>43</x>
       <y>73</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TFOC :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_distance">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>37</y>
       <width>131</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_TFoc">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>coming soon</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="arrival">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>169</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Arrival</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Busan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HochiMinh</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LongBeach</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
